docker-compose.yaml

```diff
@@ -65,3 +70,37 @@ services:
       retries: 5
       start_period: 20s
       timeout: 10s
+
+  #prometheus
+  prometheus:
+    image: prom/prometheus
+    ports:
+      - "9090:9090"
+    volumes:
+      - ./prometheus.yml/prometheus.yml:/etc/prometheus/prometheus.yml
+      - prometheus_data:/prometheus
+    command:
+      - '--config.file=/etc/prometheus/prometheus.yml'
+      - '--storage.tsdb.path=/prometheus'
+
+  #loki
+  loki:
+    image: grafana/loki:latest
+    ports:
+      - "3100:3100"
```

Comment:
Please be consistent with syntax. Decide whether you want to use single and double quotation marks.